bionic/util/__init__.py

```diff
@@ -0,0 +1,8 @@
+"""
```

Comment:
How do you feel about calling it util**s**?

Reply:
I feel good about it!